src/Symfony/Component/DependencyInjection/Compiler/PassConfig.php

```diff
@@ -52,14 +52,14 @@ public function __construct()
             new ValidateEnvPlaceholdersPass(),
             new ResolveChildDefinitionsPass(),
             new RegisterServiceSubscribersPass(),
-            new DecoratorServicePass(),
-            new ResolveParameterPlaceHoldersPass(false),
-            new ResolveFactoryClassPass(),
             new ResolveNamedArgumentsPass(),
-            new AutowireRequiredMethodsPass(),
             new ResolveBindingsPass(),
             new ServiceLocatorTagPass(),
             new CheckDefinitionValidityPass(),
+            new DecoratorServicePass(),
+            new ResolveParameterPlaceHoldersPass(false),
```

Comment:
should we let this one where it was?

Reply:
sure :+1: tests are still green after the change so I'll push it in a sec

Reply:
done